### ISSUE TITLE
Ring Lock throws an exception

### ISSUE THREAD

**Issue by dragonpil** [SeydX/homebridge-fritz-platform#145]
Hi there, 
First of all thank you very much for the new version of fritz-platform. I upgraded to 5.0.20 and removed in advance the old version (like recommended) I have a Fritz!Box 5640 (Fiber) and I have the extra Ring Lock feature as a switch. 

This switch appears always on in my home.app and also in home bridge. When I try to deactivate it, I get the following stack trace:
```
[12/23/2020, 11:46:01 AM] [FritzPlatform] FritzBox RingLock: An error occured during setting state (ringlock)
[12/23/2020, 11:46:01 AM] [FritzPlatform] TypeError: content.forEach is not a function
    at parse (/homebridge/node_modules/homebridge-fritz-platform/node_modules/cheerio/lib/parse.js:38:15)
    at Function.exports.load (/homebridge/node_modules/homebridge-fritz-platform/node_modules/cheerio/lib/static.js:39:14)
    at Object.parseOutput (/homebridge/node_modules/homebridge-fritz-platform/src/helper/lua.js:54:21)
    at Object.request (/homebridge/node_modules/homebridge-fritz-platform/src/helper/lua.js:18:21)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at ExtrasSwitchAccessory.set (/homebridge/node_modules/homebridge-fritz-platform/src/helper/deviceHandler.js:2241:13)

```
In the previous version (4.x.x) I used this in combination with good night and good morning szean to activate / deactivate the ring lock when I go to sleep. am I doing something wrong here? 

I tried to configure it like this
```
                "ringlock": {
                    "active": true,
                    "DECTphones": 1,
                    "start": "00:00",
                    "end": "23:59",
                    "accType": "switch"
                },
```
and also tried it with end 00:01

**Comment by SaschaDe** [comment]
Hi, 
I got the same error message:

> [12/29/2020, 8:03:05 PM] [FritzPlatform] FRITZ!Box RingLock: An error occured during setting state (ringlock)
[12/29/2020, 8:03:05 PM] [FritzPlatform] TypeError: content.forEach is not a function
    at parse (/usr/local/lib/node_modules/homebridge-fritz-platform/node_modules/cheerio/lib/parse.js:38:15)
    at Function.exports.load (/usr/local/lib/node_modules/homebridge-fritz-platform/node_modules/cheerio/lib/static.js:39:14)
    at Object.parseOutput (/usr/local/lib/node_modules/homebridge-fritz-platform/src/helper/lua.js:54:21)
    at Object.request (/usr/local/lib/node_modules/homebridge-fritz-platform/src/helper/lua.js:18:21)
    at processTicksAndRejections (node:internal/process/task_queues:93:5)
    at ExtrasSwitchAccessory.set (/usr/local/lib/node_modules/homebridge-fritz-platform/src/helper/deviceHandler.js:2241:13)

My config:

> "ringlock": {
                    "active": true,
                    "DECTphones": 1,
                    "start": "00:00",
                    "end": "23:59",
                    "accType": "switch"
                },

I own a Fritz!Box 5490 with Fritz!OS 07.12.

**Comment by SeydX** [comment]
Will check this, thx!

**Comment by SeydX** [comment]
Should be fixed with v5.0.21

**Comment by dragonpil** [comment]
Retested. Works now fine, like a charm.
